internal/settings/settings.go

```diff
@@ -46,6 +49,18 @@ func (o *Options) Validate() error {
 		}
 	}
 
+	if len(o.IgnoreDirectoryNames) > 0 {
+		for _, directory := range o.IgnoreDirectoryNames {
+			if directory == datadir.DataDirName {
+				return fmt.Errorf("cannot ignore data directory %q", datadir.DataDirName)
```

Comment:
```suggestion
				return fmt.Errorf("cannot ignore directory %q", datadir.DataDirName)
```

Reply:
AFAIK we never refer to that directory as "data directory" in any user-facing docs, this is just internal name, so I'd keep it simpler.